### PR TITLE
refactor

### PR DESCRIPTION
**Pull request by oliachipko**
splited template to separate components

### CODE REVIEW

**Review by rostikdoroshenko — APPROVED** [review]

**Review by YuriShevchuk — APPROVED** [review]

**Review by roksolanadutchak — APPROVED** [review]

File: src/app/ubs-admin/components/ubs-admin-address-details/ubs-admin-address-details.component.ts [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+
+@Component({
+  selector: 'app-ubs-admin-address-details',
+  templateUrl: './ubs-admin-address-details.component.html',
+  styleUrls: ['./ubs-admin-address-details.component.scss']
+})
+export class UbsAdminAddressDetailsComponent implements OnInit {
+  constructor() {}
+
+  ngOnInit() {}
```
Comment (JimHolman1199) [inline src/app/ubs-admin/components/ubs-admin-address-details/ubs-admin-address-details.component.ts:11]: let's remove an empty constructor and ngOnInit

**Review by Edw1x — APPROVED** [review]

File: src/app/ubs-admin/components/ubs-admin-address-details/ubs-admin-address-details.component.html [hunk 2]
```diff
@@ -0,0 +1,43 @@
+<div class="container mb-3">
```
Comment (Edw1x) [inline src/app/ubs-admin/components/ubs-admin-address-details/ubs-admin-address-details.component.html:1]: Change PR title

**Review by flypsoff — APPROVED** [review]

**Review by andreikazmirovich — APPROVED** [review]

File: src/app/main/component/eco-news/components/create-edit-news/create-edit-news.component.spec.ts [hunk 3]
```diff
@@ -1,34 +1,36 @@
...
-import { ACTION_CONFIG, ACTION_TOKEN } from './action.constants';
-import { MatSnackBarComponent } from '@global-errors/mat-snack-bar/mat-snack-bar.component';
-import { EcoNewsService } from '@eco-news-service/eco-news.service';
-import { of } from 'rxjs';
-import { EcoNewsModel } from '@eco-news-models/eco-news-model';
 import { FormBuilder } from '@angular/forms';
-import { CreateEditNewsFormBuilder } from './create-edit-news-form-builder';
-import { CreateEcoNewsService } from '@eco-news-service/create-eco-news.service';
-import { Router } from '@angular/router';
 import { CUSTOM_ELEMENTS_SCHEMA } from '@angular/core';
```
Comment (JimHolman1199) [inline src/app/main/component/eco-news/components/create-edit-news/create-edit-news.component.spec.ts:8]: why do we have this files here ?

File: src/app/main/component/eco-news/components/create-edit-news/create-edit-news.component.spec.ts [hunk 4]
```diff
@@ -363,4 +367,108 @@ xdescribe('CreateEditNewsComponent', () => {
...
+
+  it('should test form validating', () => {
+    const form = component.form;
+    const contentInput = form.controls.content;
+    const titleInput = form.controls.title;
+    const invalidString = `Lorem ipsum dolor sit amet, consectetur adipisicing elit. Ab dicta doloremque illum libero
+     nihil recusandae tempora veniam? Ad architecto aspernatur consectetur consequuntur culpa cupiditate dignissimos
+     dolorem ea earum enim error eum inventore ipsam ipsum iure laborum laudantium maxime modi molestiae natus nobis
+     numquam odit optio quae quaerat quas quidem rem repellat repellendus repudiandae, sequi sint sit ullam vel vero
+     voluptate voluptatem! Ab, cum et harum quas repellendus saepe voluptatem. Distinctio dolorem molestiae mollitia
```
Comment (JimHolman1199) [inline src/app/main/component/eco-news/components/create-edit-news/create-edit-news.component.spec.ts:400]: why do we have 75 files in this PR?

**Review by JimHolman1199 — CHANGES_REQUESTED** [review]

File: src/app/main/component/eco-news/components/create-edit-news/create-edit-news.component.spec.ts (hunk 4 above)
Comment (oliachipko) [reply src/app/main/component/eco-news/components/create-edit-news/create-edit-news.component.spec.ts:400]: @JimHolman1199 oliachipko
Merge branch 'dev' into base-feature/#2795-ubs-admin-cabinet
we have these files as the result of merging develop into our branch to have our branch up-to-date with develop...

**Comment by JimHolman1199** [comment]
please fix comments about PR title and an empty constructor and open it again

**Pull request closed without merging**